### ISSUE TITLE
Accessibility: Mini Cart footer elements sticky on zoom and block content in cart

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43639]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When zooming the website in, the Mini Cart has sticky elements in the footer that block being able to view what is in the cart. Here is a example which zoomed in 400%:
<img width="1579" alt="Screenshot of mini cart sticky items" src="https://github.com/woocommerce/woocommerce/assets/13153456/384850ba-765a-41bd-90ea-6f820717d7ce">

### Expected behavior

Low-vision users frequently zoom in on websites to be able to use them. When zooming the website there should be no sticky elements or elements blocking user's ability to read the content.

### Actual behavior

Sticky elements in the Mini Cart block users from being able to see what is in the cart.

### Steps to reproduce

1. Open a browser window on a website and zoom your bowser to 250% or greater.
2. Add a product to the cart and expand the mini cart.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
